src/sections/Community/Handbook/Handbook.style.js

```diff
@@ -73,6 +73,22 @@ h2 h3{
         }
     }
     
+    table {
+     border-collapse: collapse;
+     width: 100%;
+     margin: 20px 0 30px 0;
+        }
+
+   td, th {
+     border: 1px solid #dddddd;
```

Comment:
Any color that can be used from the theme instead of this? (It's fine if you don't find any)